<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Eduard Urbach</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flua Studio 0.4.5 (21 Nov 2012)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icons/languages/flua.png</normaloff>../images/icons/languages/flua.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="mainMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionReopenLastFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuModule">
    <property name="title">
     <string>Module</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
    <addaction name="actionRunOptimized"/>
    <addaction name="actionRunDebug"/>
    <addaction name="actionRunModuleTests"/>
    <addaction name="separator"/>
    <addaction name="actionCleanAllTargets"/>
    <addaction name="actionViewSource"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionChangeLog"/>
    <addaction name="actionDownloadUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionResetLocalChanges"/>
    <addaction name="actionResetLocalFileChanges"/>
    <addaction name="separator"/>
    <addaction name="actionReportBug"/>
    <addaction name="separator"/>
    <addaction name="actionThanksTo"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionRegExSearch"/>
    <addaction name="actionFindNext"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Workspaces</string>
    </property>
    <addaction name="actionWorkspace_1"/>
    <addaction name="actionWorkspace_2"/>
    <addaction name="actionWorkspace_3"/>
    <addaction name="actionWorkspace_4"/>
    <addaction name="separator"/>
    <addaction name="actionWorkspace_Q"/>
    <addaction name="actionWorkspace_W"/>
    <addaction name="actionWorkspace_E"/>
    <addaction name="actionWorkspace_R"/>
   </widget>
   <widget class="QMenu" name="menuRepository">
    <property name="title">
     <string>Repository</string>
    </property>
    <addaction name="actionRepositoryList"/>
    <addaction name="actionConnectWithGitHub"/>
   </widget>
   <widget class="QMenu" name="menuUtilities">
    <property name="title">
     <string>Utilities</string>
    </property>
    <addaction name="actionCommand"/>
    <addaction name="separator"/>
    <addaction name="actionJumpToDefinition"/>
    <addaction name="separator"/>
    <addaction name="actionCreateDefaultImplementation"/>
    <addaction name="actionDuplicateLine"/>
    <addaction name="actionToggleComment"/>
    <addaction name="separator"/>
    <addaction name="actionFindPossibleParallelizationPoints"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionToggleFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuEnvironment">
    <property name="title">
     <string>Environments</string>
    </property>
    <addaction name="actionEnvFlua"/>
    <addaction name="actionEnvPython"/>
    <addaction name="actionEnvCPP"/>
    <addaction name="actionEnvGLSL"/>
    <addaction name="separator"/>
    <addaction name="actionEnvNone"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuUtilities"/>
   <addaction name="menuModule"/>
   <addaction name="menuRepository"/>
   <addaction name="menuWindows"/>
   <addaction name="menuEnvironment"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/document-new.png</normaloff>../images/icons/actions/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/system-log-out.png</normaloff>../images/icons/actions/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/document-open.png</normaloff>../images/icons/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/apps/help-browser.png</normaloff>../images/icons/apps/help-browser.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/media-playback-start.png</normaloff>../images/icons/actions/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/document-save.png</normaloff>../images/icons/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/document-save-as.png</normaloff>../images/icons/actions/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-undo.png</normaloff>../images/icons/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-redo.png</normaloff>../images/icons/actions/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionViewDependencies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dependencies</string>
   </property>
  </action>
  <action name="actionViewXML">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>XML View</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/document-properties.png</normaloff>../images/icons/actions/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-copy.png</normaloff>../images/icons/actions/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-paste.png</normaloff>../images/icons/actions/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-cut.png</normaloff>../images/icons/actions/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/categories/preferences-system.png</normaloff>../images/icons/categories/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionAutoComplete">
   <property name="text">
    <string>Auto Complete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/window-close.png</normaloff>../images/icons/actions/window-close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAutoComplete_2">
   <property name="text">
    <string>Auto Complete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionThanksTo">
   <property name="text">
    <string>Thanks to...</string>
   </property>
  </action>
  <action name="actionDownloadUpdates">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/go-down.png</normaloff>../images/icons/actions/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Download updates</string>
   </property>
  </action>
  <action name="actionReopenLastFile">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/tab-new.png</normaloff>../images/icons/actions/tab-new.png</iconset>
   </property>
   <property name="text">
    <string>Reopen last file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionIntroduction">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionRunOptimized">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/media-seek-forward.png</normaloff>../images/icons/actions/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>Run optimized</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionRunDebug">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/docks/debugger.png</normaloff>../images/icons/docks/debugger.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionToggleFullscreen">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/view-fullscreen.png</normaloff>../images/icons/actions/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Toggle fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionChangeLog">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/apps/accessories-text-editor.png</normaloff>../images/icons/apps/accessories-text-editor.png</iconset>
   </property>
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
  <action name="actionRun_module_test">
   <property name="text">
    <string>Run module test</string>
   </property>
  </action>
  <action name="actionRunModuleTests">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/start-here.png</normaloff>../images/icons/places/start-here.png</iconset>
   </property>
   <property name="text">
    <string>Run module tests</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionCleanAllTargets">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-clear.png</normaloff>../images/icons/actions/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clean all targets</string>
   </property>
  </action>
  <action name="actionResetLocalChanges">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-clear.png</normaloff>../images/icons/actions/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Reset all local changes</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-find.png</normaloff>../images/icons/actions/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRegExSearch">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-find.png</normaloff>../images/icons/actions/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>RegEx Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/docks/debugger.png</normaloff>../images/icons/docks/debugger.png</iconset>
   </property>
   <property name="text">
    <string>Submit a Bug Report</string>
   </property>
  </action>
  <action name="actionRepositoryList">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/apps/internet-web-browser.png</normaloff>../images/icons/apps/internet-web-browser.png</iconset>
   </property>
   <property name="text">
    <string>Repository list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionConnectWithGitHub">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/apps/github.png</normaloff>../images/icons/apps/github.png</iconset>
   </property>
   <property name="text">
    <string>Connect with GitHub</string>
   </property>
  </action>
  <action name="actionResetLocalFileChanges">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/actions/edit-clear.png</normaloff>../images/icons/actions/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Reset changes for this module</string>
   </property>
  </action>
  <action name="actionFAQUpdate">
   <property name="text">
    <string>Why does updating fail?</string>
   </property>
  </action>
  <action name="actionFAQCompiling">
   <property name="text">
    <string>Why does the compiler not work?</string>
   </property>
  </action>
  <action name="actionViewSource">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/mimetypes/text-x-generic.png</normaloff>../images/icons/mimetypes/text-x-generic.png</iconset>
   </property>
   <property name="text">
    <string>View generated source code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFAQTabs">
   <property name="text">
    <string>Why can't I use spaces instead of tabs?</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionCreateDefaultImplementation">
   <property name="text">
    <string>Create default implementation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionJumpToDefinition">
   <property name="text">
    <string>Jump to definition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionDuplicateLine">
   <property name="text">
    <string>Duplicate line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionToggleComment">
   <property name="text">
    <string>Toggle comment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionFindPossibleParallelizationPoints">
   <property name="text">
    <string>Find possible parallelization points</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>Find next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionCommand">
   <property name="text">
    <string>Command</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionWorkspace_1">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace 1</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionWorkspace_2">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace 2</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionWorkspace_3">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace 3</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionWorkspace_4">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace 4</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionWorkspace_Q">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace Q</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionWorkspace_W">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace W</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="actionWorkspace_E">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace E</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionWorkspace_R">
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/places/user-desktop.png</normaloff>../images/icons/places/user-desktop.png</iconset>
   </property>
   <property name="text">
    <string>Workspace R</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionHideMainMenu">
   <property name="text">
    <string>Hide main menu</string>
   </property>
  </action>
  <action name="actionEnvNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionEnvFlua">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/languages/flua.png</normaloff>../images/icons/languages/flua.png</iconset>
   </property>
   <property name="text">
    <string>Flua</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEnvPython">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/languages/python.png</normaloff>../images/icons/languages/python.png</iconset>
   </property>
   <property name="text">
    <string>Python 3</string>
   </property>
  </action>
  <action name="actionEnvCPP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icons/languages/c++.png</normaloff>../images/icons/languages/c++.png</iconset>
   </property>
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionEnvGLSL">
   <property name="text">
    <string>GLSL</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
